lib/ReactTags.js

```diff
@@ -56,6 +56,42 @@ class ReactTags extends React.Component {
     this.setState({ query })
   }
 
+  handlePaste (e) {
+    // allow over-ride, if there is a need
+    if (this.props.onPaste) {
+      return this.props.onPaste(e)
```

Comment:
the general React pattern is to call handlers `onXYZ` though? (as this is the keyword that gets passed in through JSX's `<ReactTags onPaste={...} />`